config/email/email.js

```diff
@@ -0,0 +1,116 @@
+const nodemailer = require('nodemailer');
+const env = process.env.NODE_ENV || 'development';
+const config = require('../../dbconfig')[env];
+const CUSTOM_ENUMS = require('../../utils/enums');
+var moment = require('moment'); //datetime
+const uuidv4 = require('uuid/v4');
+var initModels = require('../../models/init-models');
+var sequelise = require('../../config/db/db_sequelise');
+const { clearConfigCache } = require('prettier');
+var models = initModels(sequelise);
+
+//emailer configuration
+// Testing Emails Pattern
+// when testing emails, in NODE_ENV=development, set EMAIL_OVERRIDE
+// if EMAIL_OVERRIDE is set, send email to it's value, prepend subject line with [TEST EMAIL], include intended recipients in the body
+const emailTransport = nodemailer.createTransport({
+  host: process.env.EMAIL_HOST,
+  port: process.env.EMAIL_PORT,
+  auth: {
+    user: process.env.EMAIL_ADDRESS,
+    pass: process.env.WEBAPP_PASSWORD,
+  },
+});
+
+// test email connection and authentication
+console.log('Checking email connection and authentication');
+emailTransport
+  .verify()
+  .then(console.log('Success - email connects and authenticates.'))
+  .catch(console.error);
+
+const customSendEmail = function (recipient, subject, body) {
+  //check var
+  const toCheck = () => {
+    return process.env.NODE_ENV == 'development' ? '' + process.env.EMAIL_OVERRIDE : recipient;
+  };
+  const subjectCheck = () => {
+    return process.env.NODE_ENV == 'production'
```

Comment:
@DanielmAaron 

import `var CUSTOM_ENUMS = require('./utils/enums');` (similar to `server.js`

then `process.env.NODE_ENV == CUSTOM_ENUMS.PRODUCTION` instead of hardcoding `'production'`